Makefile_packaging.mk

```diff
@@ -0,0 +1,269 @@
+# Common Makefile for including
+# Needs the following variables set at a minium:
+# NAME :=
+# DEB_NAME :=
+# SRC_EXT :=
+# SOURCE =
+# ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
+
+ifeq ($(DEB_NAME),)
+DEB_NAME := $(NAME)
+endif
+
+# Find out what we are
+ID_LIKE := $(shell . /etc/os-release; echo $$ID_LIKE)
```

Comment:
`lsb_release` is the portable way of discovering what distro/version/release/etc. you are on rather than the hodge-podge of various distro-specific methods.

Reply:
I will look at that as a future enhancement.  I need to collect sample output from the Leap containers.